.github/workflows/lte-integ-test-bazel.yml

```diff
@@ -23,7 +23,8 @@ on:
       - completed
 
 env:
-  CACHE_KEY: magma-dev-vm
+  CACHE_KEY_MAGMA_DEV: magma-dev-vm
+  CACHE_KEY_MAGMA_TEST: magma-test-vm
```

Comment:
do we expect different caches to the dev setup? could we use the caches from magma-dev-vm builds?

Reply:
There is some compatibility, but this build is on the `magma_test` VM which is a different environment, so it might be safer to separate the caches.

Reply:
I realize that it's a different VM - are the concerns legitimated? in terms of reusing resources, I would prefer to use the same cache if possible.

Reply:
Discussed offline - follow up ticket to consolidate the cache keys https://github.com/magma/magma/issues/13733